pandas/core/groupby/generic.py

```diff
@@ -261,7 +261,7 @@ def aggregate(self, func=None, *args, **kwargs):
 
             try:
                 return self._python_agg_general(func, *args, **kwargs)
-            except AssertionError:
+            except (AssertionError, AttributeError):
```

Comment:
what hits AttributeError here?

Reply:
we have tests that raise AttributeError here (in master) when a Categorical or IntegerArray is passed to SeriesBinGrouper.